.config-schema.json

```diff
@@ -409,7 +409,7 @@
                     "type": "object",
                     "properties": {
                         "endpoint": {
-                            "description": "The grpc endpoint of the trace collector",
+                            "description": "The grpc endpoint of the trace collector. 'none' for disabling the exporter",
```

Comment:
@adriantam I wonder if our assumption of a good default for this(and other properties) should be challenged instead. As in, you explicitly enable tracing + explicitly configure `OTLPTraceConfig.Endpoint.

Reply:
@miparnisari thoughts?

Reply:
I actually thought implementing it that way first (i.e enable exporter if endpoint not empty), but that would be a backward incompatible change, so thought that it would be better to add a keyword for disabling it. 

Do you want me to update it or keep the PR as it is?

Reply:
So, is the future plan like this?
```go
switch {
case trace-otlp-endpoint == "none":
    // enable tracing, but do not enable exporter
case trace-otlp-endpoint != "":
    // enable tracing and enable exporter with trace-otlp-endpoint endpoint
default:
   // empty endpoint, do not enable tracing
}
```

I think it would be more intuitive to keep `enabled` and only enable exporter like this:
```go
if trace-enabled {
    // enable tracing
    if trace-otlp-endpoint != "" {
       // enable exporter with trace-otlp-endpoint endpoint
    }
}
```

In order to update the `config.schema`, which path should I assume for the future plans?